daml-lf/language/daml-lf.bzl

```diff
@@ -10,8 +10,7 @@
 lf_version_configuration = {
     "legacy": "1.8",
     "stable": "1.12",
-    "latest": "1.12",
-    "preview": "1.13",
+    "latest": "1.13",
```

Comment:
Not really sure of what I am doing here.

Reply:
I think `stable` is the compiler default and `latest` is stable but not default? We really need some comments in this file and perhaps also more sensible names.